scripts/src/primazactl/primazaworker/primazaworker.py

```diff
@@ -0,0 +1,27 @@
+
+from urllib.parse import urlparse
+from primazactl.kubeconfigcontent import KubeConfigContent
+
+
+class PrimazaWorker(object):
+
+    worker_namespace: str = None
+    kube_config_file: str = None
+    worker_cluster_name: str = None
+
+    def __init__(self, cluster_name: str,
+                 kubeconfigfile: str,
+                 namespace: str = "primaza-system-worker"):
+        print("Init called")
+        self.worker_cluster_name = cluster_name
+        self.kube_config_file = kubeconfigfile
+        self.worker_namespace = namespace
+
+    def update_kube_config(self, kubeconfigfile: str):
+        kube_config = KubeConfigContent(self.primaza_cluster_name,
+                                        kubeconfigfile)
+        server_url = kube_config.get_server_url()
+        url = urlparse(server_url)
+        new_url = url._replace(netloc=url.hostname+':6443').geturl()
```

Comment:
`6443` is the port on which kind's kubernetes API Server is exposed, but other kubernetes implementation are using different ones, like minikube is using 8443. As it is dependent on the implementation, I think we should assume the kubeconfig is provided to us as correct and we should not edit it.

Reply:
Great, I did not like that code.

Reply:
if it's needed for now, leave this code here. Otherwise I'd say we should remove it

Reply:
I deleted the file, none of it is need yet.

Reply:
deleted the file, not need at all right now.